booking-app/src/main/java/org/motechproject/bookingapp/service/VisitBookingDetailsService.java

```diff
@@ -23,4 +24,12 @@
     VisitBookingDetails createOrUpdate(VisitBookingDetails visitBookingDetails);
 
     void createOrUpdate(Collection<VisitBookingDetails> visitBookingDetailsList);
+
+    List<VisitBookingDetailsDto> getVisitDtos(int page, int pageSize, String sortColumn, String sortDirection);
```

Comment:
I think this logic should be move to separate service PrimeVaccinationScheduleService, @kkaczmarczyk what do you think?